gui/wxpython/datacatalog/catalog.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, parent, giface=None, id=wx.ID_ANY,
         self.parent = parent
         self.baseTitle = title
         self.giface = giface
+        self.first_time_mode = False
```

Comment:
This should be `show_info_after_import`. I think it best expresses what it does and how it is used. The new name is explicit without introducing any new concepts.